sdk/test/logs/logger_provider_sdk_test.cc

```diff
@@ -103,4 +104,30 @@ TEST(LoggerProviderSDK, GetResource)
   LoggerProvider lp{nullptr, resource};
   ASSERT_EQ(nostd::get<std::string>(lp.GetResource().GetAttributes().at("key")), "value");
 }
+
+TEST(LoggerProviderSDK, Shutdown)
+{
+  std::unique_ptr<SimpleLogProcessor> processor(new SimpleLogProcessor(nullptr));
```

Comment:
we shouldn't pass null exporter.

Reply:
This is just the same as https://github.com/open-telemetry/opentelemetry-cpp/blob/main/sdk/test/trace/tracer_provider_test.cc#L80 and https://github.com/open-telemetry/opentelemetry-cpp/blob/main/sdk/test/trace/tracer_provider_test.cc#L91 . Is there NoopExporter right now?

Reply:
We don't handle null exporter in many places. But you are right it's not new to use null in tests. We have no `NoopExporter` AFIK.